src/redux/rockets.js/rockets.js

```diff
@@ -0,0 +1,35 @@
+import SpacesxService from '../../services/SpacexService';
```

Comment:
- [ ] There is a folder naming miss match. You are trying to import from  `../../services/SpacexService`  but the directory structure is actually like this `../../services/SpaceService`